packages/widget_toolkit_biometrics/example/pubspec.yaml

```diff
@@ -0,0 +1,32 @@
+name: example
+description: An example project to present how to use widget_toolkit_biometrics package with minimal effort.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+version: 1.0.0+1
+
+environment:
+  sdk: '>=2.19.4 <3.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
+  provider:
+  flutter_rx_bloc: ^5.0.2
+  rxdart:
+  widget_toolkit: ^0.0.1-dev5
+  widget_toolkit_biometrics: ^0.0.1-dev1
+  shared_preferences: ^2.0.20
+  app_settings: ^4.2.0
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^2.0.0
+
+dependency_overrides:
```

Comment:
Please move the overrides in a separate file, as well as add the dependency_overrides.yaml to the `.pubignore`

An example
 - https://github.com/Prime-Holding/widget_toolkit/tree/develop/packages/widget_toolkit
 - https://github.com/Prime-Holding/widget_toolkit/blob/develop/packages/widget_toolkit/.pubignore